docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
Do you have a larger mailbox? You could just grep through the mails, if 
there are many with spamassassin-stuff from other mail hosts. I will try 
that as well.

On 2016-08-20 18:26, Rimas Kudelis wrote:
> In docs/configure [1]:
> 
>> @@ -492,7 +487,6 @@ dnslookup:
>>    domains = ! +local_domains
>>    transport = remote_smtp
>>    ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
>> -  headers_remove = X-Spam-Report:X-Spam-Score
> 
> go on then :)
> 
> I think I'd like to also explain how to compact X-Spam-Report to look
> like X-Spam-Status somewhere in the config/manual, cuz I really love
> that bit. At the very least message preview is totally useless in that
> header.
> 
> BTW should we remove other headers usually added by SpamAssassin from
> received messages as well?
> 
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/vexim/vexim2/pull/203#discussion_r75581248
> [2]
> https://github.com/vexim/vexim2/pull/203/files/bde6bf3db96db591e59704833a83b6462e53274e#r75581248
> [3]
> https://github.com/notifications/unsubscribe-auth/AGCFFBS0IClxDiiP0Om8pQiKrFFsFoT-ks5qhyqcgaJpZM4JoLX7